client/src/components/UserForm.js

```diff
@@ -204,7 +239,14 @@ export default function UserForm() {
           </Form.Control.Feedback>
         </div>
       </Form.Group>
-
+      <div className="w-50">
+      <ReCAPTCHA
+       sitekey="6LfUTRElAAAAAGygougsf9-TgpDcXrONCKzZGqJP"
```

Comment:
This should be stored in an application variable. (e.g. the ones that start with `REACT_APP_`).

[See doc](https://create-react-app.dev/docs/adding-custom-environment-variables/)